applications/MeshingApplication/custom_processes/metrics_levelset_process.h

```diff
@@ -301,7 +301,7 @@ class ComputeLevelSetSolMetricProcess
     /**
      * This converts the interpolation string to an enum
      * @param str: The string
-     * @param Interpolation: The equivalent enum
+     * @return Interpolation: The equivalent enum
      */
         
```

Comment:
can u put a more descriptive name? reading the mehtod i don't understand what it is about (the doc is clear)